### PR TITLE
Fluent: Add missing tests to CI

### PR DESCRIPTION
**Pull request by JasonGore**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Add Fluent CI test commands and pipeline execution. (e2e, projects, circulars)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/12038)

### CODE REVIEW

**Comment by JasonGore** [comment]
I've fixed `test:fluentui:e2e` tests, but `test:fluentui:projects` still needs fixes.

**Comment by JasonGore** [comment]
If screener job is 24 minutes, then I guess it doesn't really make sense to break out Fluent tests from Fabric tests, at least for now.

File: packages/fluentui/ability-attributes/package.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   },
   "repository": "microsoft/fluent-ui-react.git",
   "scripts": {
-    "build": "npm run schema && gulp bundle:package:no-umd",
+    "build": "yarn schema && gulp bundle:package:no-umd",
```
Comment (JasonGore) [inline packages/fluentui/ability-attributes/package.json]: TODO: If this change needs to be kept it needs to be made part of `scripts/fluent-import/index.js`.

**Comment by JasonGore** [comment]
There are random failures happening with various pipelines that I can only assume are due to ongoing [github issues](https://www.githubstatus.com/). This PR will probably have to remain open for now.

image

**Comment by kenotron** [comment]
triggering rebuild by reopening

**Pull request merged**